code/modules/photography/camera/camera_image_capturing.dm

```diff
@@ -0,0 +1,88 @@
+/obj/effect/appearance_clone
+
+/obj/effect/appearance_clone/New(loc, atom/A)			//Intentionally not Initialize().

```

Comment:
What reason could you possibly have for this?

Reply:
Didn't know if initialize was "immediate" enough to let the camera call getFlatIcon on it immediately.

Reply:
maybe you should include that in the comment